sky/engine/core/rendering/style/RenderStyle.h

```diff
@@ -627,6 +627,9 @@ class RenderStyle: public RefCounted<RenderStyle> {
     // Apple-specific property getter methods
     EPointerEvents pointerEvents() const { return static_cast<EPointerEvents>(inherited_flags._pointerEvents); }
 
+    // Flutter extension property getter methods
+    EFlutterLineBreakMode flutterLineBreakMode() const { return static_cast<EFlutterLineBreakMode>(rareInheritedData->m_flutterLineBreakMode); }
```

Comment:
I'd remove the "flutter" from these names.  We own this code now.  (For better or worse)